crates/development/Cargo.toml

```diff
@@ -5,3 +5,7 @@ version = "0.2.2"
 authors = ["David Sherret <dsherret@gmail.com>"]
 edition = "2018"
 license = "MIT"
+
+[dependencies]
+console = "0.14.0"
+similar = "0.5.0"
```

Comment:
This library looks nice. I might use it for `dprint check` as what I have going on there is madness.